vslib/inc/HostInterfaceInfo.h

```diff
@@ -38,6 +40,14 @@ namespace saivs
                     _In_ const uint8_t *buffer,
                     _In_ size_t size) const;
 
+            bool installEth2TapFilter(
+                int priority,
+                std::shared_ptr<TrafficFilter> filter);
+            bool uninstallEth2TapFilter(std::shared_ptr<TrafficFilter> filter);
```

Comment:
add empty line between each function, singler param per l line, params should be prefixed in/out